.openshift-ci/jobs/integration-tests/run-benchmarks.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+BRANCH="$(jq -r '.extra_refs[0].base_ref' <(echo "$JOB_SPEC"))"
```

Comment:
Can we use the `get_branch` function from `libs.sh` instead? https://github.com/stackrox/collector/blob/42fbb50c7f03d37bfd37ff452e2d75ca9fcca79a/.openshift-ci/scripts/lib.sh#L82